packages/viewer/src/components/viewer-pin-group/vertex-pin-group.css

```diff
@@ -0,0 +1,70 @@
+:host {
+
+  /**
+   * @prop  --viewer-annotations-pin-label-line-color: A CSS color that
+   * specifies the color of the label line
+   */
+   --viewer-annotations-pin-dot-color: var(--blue-600);
+
+  /**
+   * @prop  --viewer-annotations-pin-color: A CSS color that
+   * specifies the color of the pin
+   */
+   --viewer-annotations-pin-color: var(--blue-700);
+
+  font-size: 0.75rem;
```

Comment:
Might be worth making this value customizable as well

Reply:
Similarly, looks like there are a handful of default border + selected border colors that could be customized here in case I change the `pin-color`/`pin-dot-color` to white/black